tests/conftest.py

```diff
@@ -45,10 +48,28 @@ def mock_butler() -> Iterator[MockButler]:
     yield from patch_butler()
 
 
-@pytest.fixture
+@pytest.fixture(scope="function")
```

Comment:
function should be the default scope. Was this for clarity?

Reply:
Yeah I was actually struggling with some of the mock_butler caching based on the label.  It was really not obvious, so I felt like all the fixtures were running as some longer running fixture somehow